utils/update_build_version.py

```diff
@@ -133,7 +133,7 @@ def describe(repo_path):
     # clock time with environment variable SOURCE_DATE_EPOCH
     # containing a (presumably) fixed timestamp.
     timestamp = int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))
-    iso_date = datetime.datetime.utcfromtimestamp(timestamp).isoformat()
+    iso_date = datetime.datetime.fromtimestamp(timestamp, datetime.UTC).isoformat()
```

Comment:
`datetime.UTC` is new in 3.11. Could be replace it with something else for older version of python?

See https://docs.python.org/3/library/datetime.html#datetime.UTC.

```suggestion
    iso_date = datetime.datetime.fromtimestamp(timestamp, datetime.timezone.utc).isoformat()
```

Reply:
FYI, I believe this will fix the bazel build failure.

Reply:
Ah, yep, that's what I should have used (I didn't realize datetime.UTC was that new). I'll update to use `datetime.timezone.utc`.